SinaiStCatherine/SinaiNewEt003.xml

```diff
@@ -89,8 +89,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
               </physDesc>                                 
               <history>
                 <summary><ref target="#p1">Unit 1</ref> and <ref target="#p2">Unit 2</ref> are remains of two different manuscripts 
-                  containing the texts of the same genre (chants) and of similar dimensions, unified by chance. <ref target="#p2">Unit 2</ref> appears to be somewhat older than 
-                  <ref target="#p1">Unit 1</ref>.</summary>               
+                  containing the texts of the same genre (chants) and of similar dimensions, unified by chance.                   
```

Comment:
I would say "containing texts" without article
by "dimensions", do you mean length of text or physical dimension of the leaves? Maybe it could be clarified.
"unified by chance" - can we actually know this? Maybe a less certain way can be found to express the same sentiment, possibly by adding "possibly" or something similar.